Mathlib/MeasureTheory/Integral/Periodic.lean

```diff
@@ -221,14 +226,61 @@ protected theorem intervalIntegral_preimage (t : ℝ) (f : UnitAddCircle → E)
 
 end UnitAddCircle
 
-variable {E : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E]
+
```

Comment:
Personally, I tend to remove the blank lines before and after, in this case. Matter of taste, i.e. up to you.

Reply:
Implemented.